src/fragments/lib-v1/datastore/native_common/real-time.mdx

```diff
@@ -2,6 +2,16 @@
 
 You can subscribe to changes on your Models. This reacts dynamically to updates of data to the underlying Storage Engine, which could be the result of GraphQL Subscriptions as well as Queries or Mutations that run against the backing AppSync API if you are synchronizing with the cloud.
 
+<Callout info>
```

Comment:
This info is all correct and I know it was added in vCurrent. I might suggest removing this just to limit scope in this large PR.